test/transport/test_packet_stream.rb

```diff
@@ -978,6 +1098,9 @@ def test_enqueue_utf_8_packet_should_ensure_packet_length_is_in_bytes_and_multip
             srand(100)
             stream.client.set cipher: cipher, hmac: hmac, compression: compress
             stream.enqueue_packet(ssh_packet)
+            if PACKETS[cipher_name][hmac_name].nil? or PACKETS[cipher_name][hmac_name][compress].nil?
```

Comment:
I thinks you can remove this lines.

Reply:
Yes, that was debug code to help fill the packet test cases. Already gone in my local copy.